runtime/buffer.h

```diff
@@ -264,6 +264,8 @@ class raw_buffer {
 
   std::size_t size_bytes() const;
 
+  std::size_t elem_count() const;
```

Comment:
Could you please add a comment about what it exactly does?

Reply:
I filed #292. I think documenting this particular function in isolation is tricky, we really need to document the whole class properly in order to concisely describe what this function does. I just spent a few minutes trying to throw something together but it really depends a serious effort to do well.